service_crategen/src/commands/generate/mod.rs

```diff
@@ -1,23 +1,30 @@
+extern crate stopwatch;
+extern crate env_logger;
+
 use std::collections::BTreeMap;
 use std::fs::{self, OpenOptions};
 use std::io::{BufWriter, Write};
 use std::path::Path;
 
+use self::stopwatch::Stopwatch;
 use rayon::prelude::*;
 use rustfmt;
 use toml;
 
 mod codegen;
 
 use cargo;
+use log::LogLevel::Debug;
 use ::{Service, ServiceConfig, ServiceDefinition};
 
 pub fn generate_services(services: BTreeMap<String, ServiceConfig>, out_dir: &Path) {
+    let _ = env_logger::init(); // This initializes the `env_logger`
```

Comment:
Maybe we can change this to log when we fail to initialize the logger?

```rust
if let Err(e) = env_logger::init() {
  // Potentially write to STDERR?
  println!("Failed to initialize Logger: {:?}", e);
}
```

Reply:
That's a great change, thanks for the catch.